storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { DescriptionList } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+const meta = {
+  title: 'Components/Text/Description List',
+  component: DescriptionList,
+  args: {
+    children: [
+      <DescriptionList.Term key={1}>Gebied</DescriptionList.Term>,
+      <DescriptionList.Details key={2}>Gemeente Amsterdam</DescriptionList.Details>,
+      <DescriptionList.Term key={3}>Stadsdeel</DescriptionList.Term>,
+      <DescriptionList.Details key={4}>West</DescriptionList.Details>,
+      <DescriptionList.Term key={5}>Opmerkingen</DescriptionList.Term>,
+      <DescriptionList.Details key={6}>{paragraph}</DescriptionList.Details>,
+    ],
+    inverseColor: false,
+  },
+} satisfies Meta<typeof DescriptionList>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  decorators: [
```

Comment:
I'd add this decorator to the `meta` object. The text in the MultipleDetails story is now invisible if you set `inverseColor` to true there.

Reply:
Done